src/twig/linearize/Callable/src/bin.hpp

```diff
@@ -1,7 +1,7 @@
 template< typename Functor, typename Criterion, typename Midpoint >
 void bin( Xdata xLeft, Xdata xRight,
-	  Ydata yLeft, Ydata yRight,
-	  Functor&& functor,
+      	  Ydata yLeft, Ydata yRight,
+      	  Functor&& functor,
           Criterion&& criterion,
           Midpoint&& midpoint ){
```

Comment:
Pretty weird indentation going on here.  Looks like you have some tabs snuck in there.

Reply:
Good catch. Thanks for finding these. They should be resolved now